R/scenario_sim.R

```diff
@@ -25,22 +25,21 @@
 #' @export
 #'
 #' @examples
-#' \dontrun{
-#' res <- scenario_sim(n.sim = 5,
-#' num.initial.cases = 5,
-#' cap_max_days = 365,
-#' cap_cases = 2000,
-#' r0isolated = 0,
-#' r0community = 2.5,
-#' disp.iso = 1,
-#' disp.com = 0.16,
-#' k = 0.7,
-#' delay_shape = 2.5,
-#' delay_scale = 5,
-#' prop.asym = 0,
-#' prop.ascertain = 0)
-#' }
-#'
+#' res <- scenario_sim(
+#'   n.sim = 5,
+#'   num.initial.cases = 5,
+#'   cap_max_days = 365,
+#'   cap_cases = 2000,
+#'   r0isolated = 0,
+#'   r0community = 2.5,
+#'   disp.iso = 1,
+#'   disp.com = 0.16,
+#'   k = 0.7,
+#'   delay_shape = 2.5,
+#'   delay_scale = 5,
+#'   prop.asym = 0,
+#'   prop.ascertain = 0
+#' )
```

Comment:
how should the user look at this?

Reply:
Resolved in 5d32685e132c9a597851ad63554c86826df98776, for now with a simply printing the object. In the future these examples will need more work, but I think that major changes should take precedence.